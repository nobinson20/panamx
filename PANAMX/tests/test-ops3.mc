int main() {
    double d;
    d = -6.5 / 2;
    printf(d);
    return 0;
}
