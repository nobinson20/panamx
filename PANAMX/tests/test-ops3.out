12.375
-3.25
